samples/graphics/pdf/build.gradle.kts

```diff
@@ -26,4 +26,6 @@ android {
 
 dependencies {
     // Add samples specific dependencies
+    implementation(libs.compose.runtime.runtime)
```

Comment:
This is not needed

Reply:
Thanks. remived unnecessary dependencies